docs/user-guide/transformer_expressions.md

```diff
@@ -535,3 +549,397 @@ variables:
+
+#### Input
+No Input
+
+#### Output
+Single value with string type
+
+#### Examples
+
+Suppose users have table `yourTableName`
+
+| restaurant_id | avg_order_1_day | avg_cancellation_rate_30_day |
+|---------------|-----------------|------------------------------|
+| 1 | 2000 | 0.02 |
+| 2 | 3000 | 0.005 |
+| 3 | 4000 | 0.006 |
+
+Standard Transformer Config:
+```
+variables:
+- name: max_str
+  expression: yourTableName.Col("avg_order_1_day").MaxStr()
+```
+Output: "4000"
+
+### Min
+
+`Min` is a function to find minimum value from column values. The output will be single value in float type
+
+#### Input
+No Input
+
+#### Output
+Single value with float type
+
+#### Examples
+
+Suppose users have table `yourTableName`
+
+| restaurant_id | avg_order_1_day | avg_cancellation_rate_30_day |
+|---------------|-----------------|------------------------------|
+| 1 | 2000 | 0.02 |
+| 2 | 3000 | 0.005 |
+| 3 | 4000 | 0.006 |
+
+Standard Transformer Config:
+```
+variables:
+- name: min
+  expression: yourTableName.Col("avg_order_1_day").Min()
+```
+Output: 2000
+
+### MinStr
+
+`MinStr` is a function to find minimum value from column values. The output will be single value in string type
+
+#### Input
+No Input
+
+#### Output
+Single value with string type
+
+#### Examples
+
+Suppose users have table `yourTableName`
+
+| restaurant_id | avg_order_1_day | avg_cancellation_rate_30_day |
+|---------------|-----------------|------------------------------|
+| 1 | 2000 | 0.02 |
+| 2 | 3000 | 0.005 |
+| 3 | 4000 | 0.006 |
+
+Standard Transformer Config:
+```
+variables:
+- name: min_str
+  expression: yourTableName.Col("avg_order_1_day").MinStr()
+```
+Output: "2000"
+
+### Quantile
+
+`Quantile` is a function to returns the sample of x such that x is greater than or equal to the fraction p of samples
+
+#### Input
+Fraction in float type
+
+#### Output
+Single value with float type
+
+#### Examples
+
+Suppose users have table `yourTableName`
+
+| rank |
+|-------|
+| 1 |
+| 2 | 
+| 3 |
+| 4 |
+| 5 |
+| 6 | 
+| 7 |
+| 8 |
+| 9 |
+| 10 |
+
+Standard Transformer Config:
+```
+variables:
+- name: quantile_0.9
+  expression: yourTableName.Col("rank").Quantile(0.9)
+```
+Output: 9
+
+### Sum
+
+`Sum` is a function to sum all the values in the columns. The output will be single value in float type
+
+#### Input
+No Input
+
+#### Output
+Single value with float type
+
+#### Examples
+
+Suppose users have table `yourTableName`
+
+| restaurant_id | avg_order_1_day | avg_cancellation_rate_30_day |
+|---------------|-----------------|------------------------------|
+| 1 | 2000 | 0.02 |
+| 2 | 3000 | 0.005 |
+| 3 | 4000 | 0.006 |
+
+Standard Transformer Config:
+```
+variables:
+- name: sum
+  expression: yourTableName.Col("avg_order_1_day").Sum()
+```
+Output: 9000
+
+### Flatten
+`Flatten` is a function to flatten all values in a column, this is suitable for column that has list type, for non list the result will be the same with the original columns
```

Comment:
In a table, what is the result of flatten on the other columns? For example, after flatten on `nearby_restaurant_ids` column below, can the user select `restaurant_id` too? Will its value be preserved? Like:
```

| restaurant_id | restaurant_ids |
|---------------|-----------------|
| 1 | 2 |
| 1 | 3 |
| 1 | 4 |
| 2 | 4 |
...
```

Reply:
> In a table, what is the result of flatten on the other columns? For example, after flatten on `nearby_restaurant_ids` column below, can the user select `restaurant_id` too? Will its value be preserved? Like:
> 
> ```
> 
> | restaurant_id | restaurant_ids |
> |---------------|-----------------|
> | 1 | 2 |
> | 1 | 3 |
> | 1 | 4 |
> | 2 | 4 |
> ...
> ```

the `flatten` function will return new series (column), if you use this expression in `variables` operation, the output will be variable that type is series(column). CMIIW, your question is if user use the expression in the `updateColumns` operation like below
```
 inputTable: yourTableName
 outputTable: yourOutputTableName
  steps:
      - updateColumns:
         - column: flatten_restaurant_id
            expression: yourTableName.Col("nearby_restaurant_ids").Flatten()
```
? If that the case, the `yourOutputTableName` has the same value with the `yourTableName` and plus one column `flatten_restaurant_id` (if the dimension of new column is the same with existing table otherwise the operation will be fail)